src/main/thrift/parquet.thrift

```diff
@@ -712,9 +712,14 @@ struct DictionaryPageHeader {
 }
 
 /**
- * New page format allowing reading levels without decompressing the data
+ * Alternate page format allowing reading levels without decompressing the data
```

Comment:
I would perhaps keep "New" or "More recent".

Reply:
I think Alternate probably makes the most sense here.  It is not exactly "new" any more.

How about "Alternative page format, introduced to the specification after DataPageHeader, ...."

Reply:
the naming of "V2" somewhat implies that it came after `DataPageHeader`

What I think is confusing about the word "new" is that (to me) it implies an expectation that this header will eventually replace the existing one, which I don't think there is consensus on